packages/react/src/FAQ/FAQ.features.stories.tsx

```diff
@@ -54,26 +54,10 @@ const fixtureData: FixtureData = [
           non eros.
         </p>
         <ol>
-          <li>
-            <Text size="300" variant="muted">
-              Must be associated with a current GitHub for Startups partner.
-            </Text>
-          </li>
-          <li>
-            <Text size="300" variant="muted">
-              Self-funded or funded (Seed-Series A)
-            </Text>
-          </li>
-          <li>
-            <Text size="300" variant="muted">
-              Not a current GitHub Enterprise customer
-            </Text>
-          </li>
-          <li>
-            <Text size="300" variant="muted">
-              Must not have previously received credits for GitHub Enterprise
-            </Text>
-          </li>
+          <li>Must be associated with a current GitHub for Startups partner.</li>
```

Comment:
I noticed above you updated a list to use the UnorderedList component. Should we do that here as well?

Reply:
Would prefer to leave as-is as I'm trying to limit the number of changes to stories to only those that are necessary. Admittedly could have kept the other one the same, but also not a problem to keep it now that it's done imo.

Reply:
Sounds good to me!